root/documentation/vep.conf

```diff
@@ -133,6 +133,51 @@
         description=Use merged Ensembl and RefSeq transcript set to report consequences (human only)
         default=0
       </merged>
+      <gencode_basic>
+        type=Boolean
+        description=Limit your analysis to transcripts belonging to the GENCODE basic set. This set has fragmented or problematic transcripts removed.
+        default=0
+      </gencode_basic>
+      <pick>
+        type=Boolean
+        description=Pick one line or block of consequence data per variant, including transcript-specific columns. Consequences are chosen according to the criteria described <a target="_blank" href="http://www.ensembl.org/info/docs/tools/vep/script/vep_other.html\#pick">here</a>, and the order the criteria are applied may be customised with <b>pick_order</b>. This is the best method to use if you are interested only in one consequence per variant.
+        default=0
+      </pick>
+      <pick_allele>
+        type=Boolean
+        description=Like <b>pick</b>, but chooses one line or block of consequence data per variant allele. Will only differ in behaviour from <b>pick</b> when the input variant has multiple alternate alleles.
+        default=0
+      </pick_allele>
+      <per_gene>
+        type=Boolean
+        description=Output only the most severe consequence per gene. The transcript selected is arbitrary if more than one has the same predicted consequence. Uses the same ranking system as <b>pick</b>.
+        default=0
+      </per_gene>
+      <pick_allele_gene>
+        type=Boolean
+        description=Like <b>pick_allele</b>, but chooses one line or block of consequence data per variant allele and gene combination.
+        default=0
+      </pick_allele_gene>
+      <flag_pick>
```

Comment:
Suggesting that we add `Not used by default`,  to allow for uniformity between whats on the website and here.  Thank you

Reply:
Hi @olaaustine! Adding `Not used by default` is redundant because we show `default=0` and no argument in REST shows `Not used by default`, even if they do in the website (as in the case of `--hgvs`, `--ccds` and `--domains`).